chuffed/support/lengauer_tarjan.h

```diff
@@ -1,3 +1,5 @@
+#pragma once
```

Comment:
Good spotting that this was missing. Could you change your commit to use `#ifdef` guards to match the rest of the codebase?

Reply:
Done.